app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,17 @@
+from .dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(self,
+                 skill_level: int,
+                 nickname: str,
+                 favourite_dish: str) -> None:
+        super().__init__(favourite_dish, nickname)
+        self.skill_level = skill_level
```

Comment:
The `skill_level` attribute should be prefixed with an underscore to indicate it is protected, as per the task requirements.